### PR TITLE
replace ATOXGR and BTOXGR with ANRIND and BNRIND.

### LINKED ISSUE #68: update lbt15, use  ANRIND and BNRIND instead of AOTXGR or BTOXGR
see https://github.com/insightsengineering/chevron/pull/570#discussion_r1259636502 for more detail.

### PR DESCRIPTION
**Pull request by ayogasekaram**
closes #68

### CODE REVIEW

File: book/tables/lab-results/lbt15.qmd [hunk 1]
```diff
@@ -83,9 +87,9 @@ lyt <- basic_table(show_colcounts = TRUE) %>%
   split_rows_by("PARCAT1", split_fun = drop_split_levels, label_pos = "topleft", split_label = obj_label(adlb$PARCAT1)) %>%
   split_rows_by("PARAM", split_fun = drop_split_levels, label_pos = "topleft", split_label = obj_label(adlb$PARAM)) %>%
   count_abnormal(
-    var = "ATOXGR_CAT",
+    var = "ANRIND",
     abnormal = list(low = "LOW", high = "HIGH"),
```
Comment (edelarua) [inline book/tables/lab-results/lbt15.qmd:91]: Just a suggestion - I think this has been changed to uppercase in most tables 
```suggestion
    abnormal = list(Low = "LOW", High = "HIGH"),
```

**Review by edelarua — APPROVED** [review]
Lgtm! Just one suggestion to convert low/high labels to uppercase :)

**Review by Melkiades — CHANGES_REQUESTED** [review]
@ayogasekaram could you also update the snapshots?

**Comment by ayogasekaram** [comment]
@Melkiades there are no changes to the output table so the snapshot is the same.

**Review by Melkiades — APPROVED** [review]
Lgtm!! Thanks Abi :)

**Pull request merged**